opennms-base-assembly/src/main/filtered/etc/events/opennms.correlation.events.xml

```diff
@@ -1,26 +1,4 @@
 <events xmlns="http://xmlns.opennms.org/xsd/eventconf">
-   <event>
```

Comment:
Don't know. Maybe we want to keep such kind of event?

Reply:
Then we will recreate it...